ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.11.yaml

```
@@ -158,6 +158,18 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       TEST_ARGS: --from-repository quay.io/multi-arch/community-e2e-images
     workflow: openshift-e2e-aws
+- as: ocp-e2e-upgrade-aws-arm64
+  interval: 72h
+  steps:
+    cluster_profile: aws-arm64
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
```

Shouldn't this be
```suggestion
      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-initial
      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
```
Or will `release:initial` and `release:latest` be properly resolved to their arm64 counterparts?